inst/settings/resultsDataModelSpecification.csv

```diff
@@ -81,12 +81,12 @@ covariate_value_dist,p_25_value,float,Yes,No,No,Yes
 covariate_value_dist,p_75_value,float,Yes,No,No,Yes
 covariate_value_dist,p_90_value,float,Yes,No,No,Yes
 covariate_value_dist,database_id,varchar,Yes,Yes,No,Yes
-database,database_id,varchar,Yes,Yes,No,Yes
-database,database_name,varchar,No,No,No,Yes
-database,description,varchar,No,No,No,Yes
-database,is_meta_analysis,varchar(1),Yes,No,No,Yes
-database, vocabulary_version, varchar, No, No, Yes, Yes
-database, vocabulary_version_cdm, varchar, No, No, Yes, Yes
+database_info,database_id,varchar,Yes,Yes,No,Yes
```

Comment:
this is a breaking change

Reply:
Yes, and thinking about it we will want to support importing old zip files that contain "database.csv" files. This needs to be redisgned.